bin/install.sh

```diff
@@ -169,7 +135,7 @@ function advice {
   elif [[ "$(uname -s)" == *"Linux"* ]]; then
     echo "apt-get install $1"
   elif [[ "$(uname -s)" == *"_NT"* ]]; then
-    echo "choco install $1 ?"
+    echo "Use install.ps1 to install $1."
```

Comment:
Maybe advise the command similarly to the above variants?

Reply:
@jwerle, how about:

I'm pretty sure this is a noop, I just wanted to remove the choco reference.

I'll remove it entirely.